src/DacpacTool/Program.cs

```diff
@@ -48,7 +58,7 @@ static async Task<int> Main(string[] args)
             };
             deployCommand.Handler = CommandHandler.Create<DeployOptions>(DeployDacpac);
 
-            var rootCommand = new RootCommand { buildCommand, deployCommand };
+            var rootCommand = new RootCommand { buildCommand, inspectCommand, deployCommand };
```

Comment:
What is "inspect command"??

Reply:
New CLI command to collect the list of included files before running the build. Although we renamed the command "collect-includes", so I should rename this variable as well. I'll include that in my next update.

Reply:
@ErikEJ Do you have concerns about this, or was this just a question?

Reply:
Still a confusing name... Prefer collectIncludes or consolidateIncludes

Reply:
I believe it's now `collectIncludesCommand`

Reply:
It got renamed to collect-includes in a later commit ;).